_transactions.md

```diff
@@ -143,6 +143,9 @@ Upon preparing a transaction, a [Transaction Object](#transaction-object) will b
 <aside class="notice">
   If the deposit origin is a <code>CARD</code> account ID and the query string parameter <code>?commit=true</code> is set, you need to send the credit card's <code>securityCode</code> in the request body.
 </aside>
+<aside class="notice">
+  <strong>Important Notice</strong>: In compliance with PCI standards, the Uphold sandbox environment does not accept real credit/debit card data. For a list of accepted card data, please refer to the <a href="#cards-for-sandbox-transactions">Cards for Sandbox transactions</a> section of the documentation.
```

Comment:
The only point that I can make is the differences when using the name `sandbox` with lowercase `s` or uppercase `s`. I believe that we are using `Sandbox` most of the times, but don't know which one would be better in this instance. Nevertheless, we should choose one and use that one.

Reply:
<img width="401" alt="image" src="https://github.com/uphold/docs/assets/3464842/502ff6d0-966b-47bc-bd76-65d0496392ad">
Actually, the word sandbox doesn't show up much in the API docs, outside URLs and besides the text being currently added. When it does show up, it is mostly in the `sandbox` variant. Nevertheless, we will update all entries to `Sandbox`, with the obvious exception of the ones present in URLs.